Libraries/LibCore/EventLoopImplementationWindows.cpp

```diff
@@ -110,7 +111,7 @@ size_t EventLoopImplementationWindows::pump(PumpMode)
     for (auto& entry : timers)
         event_handles[index++] = entry.key.handle;
```

Comment:
Then I shouldn't call resize prior as it would fill the beginning elements with initialized or default initialized memory( not sure what AK does ). Should I grow_capacity( I assume this is the same as reserve for std::vector) or not care about the small performance impact of reallocation in this case?

Reply:
You should use `.ensure_capacity()` instead.